pkgs/data/misc/colemak-dh/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "colemak-dh";
+  version = "unstable-2022-08-07";
+
+  src = fetchFromGitHub {
+    owner = "ColemakMods";
+    repo  = "mod-dh";
+    rev = "e846a5bd24d59ed15ba70b3a9d5363a38ca51d09";
+    sha256  = "sha256-RFOpN+tIMfakb7AZN0ock9eq2mytvL0DWedvQV67+ks=";
+    sparseCheckout = "console";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/keymaps/i386/
+    gzip -r .
+    cp -r ./console $out/share/keymaps/i386/colemak
+  '';
```

Comment:
1. Please `runHook preInstall` and `runHook postInstall` here.
2. I think you should change the `gzip` call to `gzip -r ./console` or similar, since I don't think we want to spend the "time" gzipping the README and license 🙂 (Git sparse checkouts always include the top level files, in addition to the directories selected to sparse checkout).